.travis.yml

```diff
@@ -1,20 +1,24 @@
 language: c
 
+# Default job
+os: linux
+sudo: required
+dist: trusty
+# Should spawn a linux-gcc job and a linux-clang job
+compiler: 
+  - gcc
+  - clang
+
+# Aditional testing on OSX with default compiler: should spawn one job
 matrix:
   include:
-    - os: linux
-      sudo: required
-      dist: trusty
-      group: edge
     - os: osx
+      env: COMPILER=clang
```

Comment:
@diaena Please carefully read the following page.
https://docs.travis-ci.com/user/languages/c/

Do you have Intel compiler installed at your environment?
Support for Intel compiler was added following requests from the users.
Even though they can use SVML, they still want to use SLEEF.